experimental/rockbuilder/projects/pytorch.cfg

```diff
@@ -2,34 +2,35 @@
 name=pytorch
 repo_url=https://github.com/pytorch/pytorch.git
 version=v2.7.0
-patch_dir=patches/pytorch
-clean_cmd = #!/usr/bin/bash
-            set -e
-            cd $ROCK_BUILDER_APP_SRC_DIR
-	        python3 setup.py clean
-configure_cmd = #!/usr/bin/bash
-                set -e
-                cd $ROCK_BUILDER_APP_SRC_DIR
-	            python3" tools/amd_build/build_amd.py
-build_cmd = #!/usr/bin/bash
-            set -e
-            export PATH=${ROCM_HOME}/bin:$PATH
-            export LD_LIBRARY_PATH=${ROCM_HOME}/lib
-            export CMAKE_PREFIX_PATH="$(realpath ${ROCM_HOME})"
-            export DEVICE_LIB_PATH=${CMAKE_PREFIX_PATH}/lib/llvm/amdgcn/bitcode
-            export HIP_DEVICE_LIB_PATH=${DEVICE_LIB_PATH}
-            cd $ROCK_BUILDER_APP_SRC_DIR
-	        USE_KINETO=OFF PYTORCH_BUILD_VERSION=2.7.0 PYTORCH_BUILD_NUMBER=1 python3 setup.py bdist_wheel
+
+# common env variables both for linux and windows
+env_common=
+       USE_ROCM=1
+       USE_CUDA=0
+	   USE_KINETO=0
+       CMAKE_PREFIX_PATH=${ROCM_HOME}
+       DEVICE_LIB_PATH=${ROCM_HOME}/lib/llvm/amdgcn/bitcode
+       HIP_CLANG_PATH=${ROCM_HOME}/lib/llvm/bin
+       HIP_DEVICE_LIB_PATH=${DEVICE_LIB_PATH}
+	   PYTORCH_BUILD_VERSION=2.7.0
+	   PYTORCH_BUILD_NUMBER=1
```

Comment:
This file has tabs mixed with spaces, so the indentation is inconsistent here. I'll add a pre-commit rule banning tabs.

Reply:
https://github.com/ROCm/TheRock/pull/836 (can go in before or after your changes, will have merge conflicts either way I think)